assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -321,7 +326,14 @@ export const baseControls = {
 	} ),
 	[ SELECT_MODULE_REAUTH_URL ]: createRegistryControl( ( { select } ) => ( { payload } ) => {
 		const { slug } = payload;
-		const getAdminReauthURL = select( `modules/${ slug }` )?.getAdminReauthURL;
+		const storeName = select( 'core/modules' )?.getModuleStoreName( slug );
```

Comment:
Please use the core store name constant instead. Also, no conditional chaining needed here since we can rely on the presence of the core selector.

Reply:
This comment is yet to be addressed.